catapult-sdk/src/plugins/namespace.js

```diff
@@ -65,11 +65,19 @@ const namespacePlugin = {
 			namespaces: { type: ModelType.array, schemaName: 'namespaceDescriptor' }
 		});
 
+		builder.addSchema('namespaceDescriptorWithId', {
```

Comment:
i'm not sure i understand this...

why do we need both `namespaceDescriptor` and `namespaceDescriptorWithId`? both endpoints using those schemas should return `id`, `meta`, and `namespace`, right? it is possible that `/namespaces/:namespaceId` is not parsing the id right now

related: `/namespaces/:namespaceId` is using `findNamesByIds` which uses `sanitizer.deleteIds`, not sure this is correct

Reply:
You were right, id in the `/namespaces/:namespaceId` endpoint wasn't being parsed. Now this is fixed. Regarding `sanitizer.deleteIds` it was changed to `sanitizer.renameId` in a previous commit.